HaxeManual/08-compiler-features.tex

```diff
@@ -363,7 +363,7 @@ \subsection{Type path completion}
 <i n="run">
   <t>v : String -&gt; Dynamic</t>
   <d>
-    Unserializes `v` and returns the according value.
+    Unserializes `v` and returns according to value.
```

Comment:
The source for this is the API documentation: https://api.haxe.org/v/development/haxe/Unserializer.html

`according` should be replaced with another word I guess (`corresponding`?), but your suggestion is also incorrect.